src/riot-rs-runqueue/proofs/fstar/extraction/Riot_rs_runqueue.Runqueue.Clist.fst

```diff
@@ -0,0 +1,216 @@
+module Riot_rs_runqueue.Runqueue.Clist
```

Comment:
these two .fst files, they are generated, right? they don't have to get checked in?

Reply:
yes, you're right, those are generated, I can add them to gitignore